iOS/MasterFeed/MarkArticlesReadAlertController.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  MarkArticlesReadAlertControllerr.swift
+//  NetNewsWire
+//
+//  Created by Phil Viso on 9/29/19.
+//  Copyright © 2019 Ranchero Software. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+struct MarkArticlesReadAlertController {
+		
+	static func allArticlesAlert(handler: @escaping (UIAlertAction) -> Void) -> UIAlertController {
+		let message = NSLocalizedString("Mark all articles in all accounts as read? You can undo this action with a three finger swipe to the left.",
```

Comment:
Open to ideas about how to make the message also convey that NNW supports undo on most actions in the app. The message is already pretty long, so I was trying to keep it brief.

Reply:
I'm concerned that the information about the undo is in the second sentence and the button still uses the Mark All Read.  I think people will just hit read that button, hit it, and never read the verbiage.

Maybe we should just have a dialog that says, "You can undo this and other actions with a three finger swipe to the left." with a "Got It" button.  My hope is that the "Got It" will give them some pause and make them read the verbaige.